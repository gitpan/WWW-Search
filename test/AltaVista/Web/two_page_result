http://www.altavista.digital.com/cgi-bin/query?what=web&text=yes&fmt=d&pg=q&q=%2BLSAM+%2BISI+%2BIB
/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k
