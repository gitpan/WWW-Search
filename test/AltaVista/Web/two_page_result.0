<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +work</title>
</head><body>
<a href="/cgi-bin/query?text=yes"><b>AltaVista Search</b></a><br>

<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>

<a href="http://babelfish.altavista.digital.com/cgi-bin/translate?">Translations</a> .
<a href="http://altavista.iname.com/">Free eMail</a> .
<a href="http://altavista.looksmart.com/altavista.html">Browse by Subject</a> . 
<a href="http://altavista.switchboard.com/bin/cgiqa.dll?MG=&MEM=212">People Search</a> . 
<a href="http://altavista.switchboard.com/bin/cgidir.dll?MG=&MEM=212">Business Search</a>
<br>
<font size=-1 face=arial><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cz >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><INPUT NAME=q size=65 maxlength=800 wrap=virtual VALUE="+LSAM +ISI +work"><br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_simple.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=q&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=aq&text=yes">Advanced Search</a><br>Click here to find related books at <a href="http://www.amazon.com/exec/obidos/av-search/q=%2bLSAM+%2bISI+%2bwork">amazon.com</a>
<br>




<b>47</b> documents match your query.<dl>
<dt><b>1. </b><a href="http://www.isi.edu/~ahughes/projectweb/ib/"><b>About IB</b></a><dd>Intelligent Bandwidth. &nbsp; About Intelligent Bandwidth. &nbsp; Research Projects. &nbsp; People. &nbsp; Related Work. &nbsp; Page maintainer: Joe Touch.<br><i>http://www.isi.edu/~ahughes/projectweb/ib/ - size 2K - 14-Oct-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7eahughes%2fprojectweb%2fib%2f&language=en">Translate</a></i><br>
<P><dt><b>2. </b><a href="http://gost.isi.edu/people/johnh/SOFTWARE/"><b>Software Written by John Heidemann</b></a><dd>Software Written by John Heidemann. Writing software is both my job and one of my hobbies. The following packages are currently available: Operating...<br><i>http://gost.isi.edu/people/johnh/SOFTWARE/ - size 8K - 15-Aug-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2fpeople%2fjohnh%2fSOFTWARE%2f&language=en">Translate</a></i><br>
<P><dt><b>3. </b><a href="http://www.isi.edu/~johnh/SOFTWARE/"><b>Software Written by John Heidemann</b></a><dd>Software Written by John Heidemann. Writing software is both my job and one of my hobbies. The following packages are currently available: Operating...<br><i>http://www.isi.edu/~johnh/SOFTWARE/ - size 9K - 12-Dec-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7ejohnh%2fSOFTWARE%2f&language=en">Translate</a></i><br>
<P><dt><b>4. </b><a href="http://gost.isi.edu/~johnh/WORK/"><b>John Heidemann's Work</b></a><dd>John Heidemann's Work. I am computer scientist at Information Sciences Institute and a research assistant professor at the University of Southern...<br><i>http://gost.isi.edu/~johnh/WORK/ - size 6K - 14-Nov-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2f%7ejohnh%2fWORK%2f&language=en">Translate</a></i><br>
<P><dt><b>5. </b><a href="http://www.cs.arizona.edu/xkernel/www/ipsec/ipsec-mail4.html"><b>Mailing list archive ipsec-mail4</b></a><dd>Mailing list archive ipsec-mail4. From ipsec-request@neptune.tis.com Tue Mar 19 06:39:26 1996 X-Sender: t3125rm@pop3hub.is.chrysler.com X-Mailer: Windows..<br><i>http://www.cs.arizona.edu/xkernel/www/ipsec/ipsec-mail4.html - size 577K - 30-Jan-98 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2ecs%2earizona%2eedu%2fxkernel%2fwww%2fipsec%2fipsec%2dmail4%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>6. </b><a href="http://www.wls.org/resume.html"><b>wls' resume @ moviola.com</b></a><dd>William L. Scheding. 12301 Clover Ave. Los Angeles, CA, 90066-1020 work: 213.467.3107 -- home: 310.398.6502 -- pager: 310.775.6502 bill@moviola.com....<br><i>http://www.wls.org/resume.html - size 9K - 10-Sep-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fwww%2ewls%2eorg%2fresume%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>7. </b><a href="http://boggy.epistat.m.u-tokyo.ac.jp:6080/=@=:www2.umin.u-tokyo.ac.jp/internet/drafts/1id-abstracts.txt"><b>No Title</b></a><dd>Current Internet-Drafts This summary sheet provides a short synopsis of each Internet-Draft available within the "internet-drafts" directory at the shadow.<br><i>http://boggy.epistat.m.u-tokyo.ac.jp:6080/=@=:www2.umin.u-tokyo.ac.jp/internet/drafts/1id-abstracts.txt - size 574K - 24-Apr-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fboggy%2eepistat%2em%2eu%2dtokyo%2eac%2ejp%3a6080%2f%3d%40%3d%3awww2%2eumin%2eu%2dtokyo%2eac%2ejp%2finternet%2fdrafts%2f1id%2dabstracts%2etxt&language=en">Translate</a></i><br>
<P><dt><b>8. </b><a href="http://ito.darpa.mil/PIMeetings/ICV-IM/agenda.html"><b>DARPA IC&V/IM PI Meeting: Agenda</b></a><dd>DARPA / ITO Joint IC&V/IM Principal Investigator Meeting October 15-17, 1997 San Diego, CA Agenda. IC&V Home Page. IM Home Page. Intro. Agenda. Who is...<br><i>http://ito.darpa.mil/PIMeetings/ICV-IM/agenda.html - size 13K - 18-Nov-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fito%2edarpa%2emil%2fPIMeetings%2fICV%2dIM%2fagenda%2ehtml&language=en">Translate</a></i><br>
<P><dt><b>9. </b><a href="http://psaweb.pisa.otm.it/cgi-bin/browse.pl/archweb/develop/httpd/apache"><b>Category :  Httpd : Apache</b></a><dd>nbsp; Httpd : Apache &#149;&#160;&#160;Manual &#149;&#160;&#160;Mod_perl &#149;&#160;&#160;Tutorial &nbsp;&nbsp;&nbsp;Page: 1 | 2. Apache Week: Apache...<br><i>http://psaweb.pisa.otm.it/cgi-bin/browse.pl/archweb/develop/httpd/apache - size 9K - 23-Jan-98 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fpsaweb%2episa%2eotm%2eit%2fcgi%2dbin%2fbrowse%2epl%2farchweb%2fdevelop%2fhttpd%2fapache&language=en">Translate</a></i><br>
<P><dt><b>10. </b><a href="http://gost.isi.edu/~johnh/SOFTWARE/APACHE/"><b>LSAM Apache Patches</b></a><dd>LSAM Apache Patches. As part of my work with ISI's LSAM project we have found some optimizations in http-server performance. We describe these problems in.<br><i>http://gost.isi.edu/~johnh/SOFTWARE/APACHE/ - size 3K - 1-Jul-97 - English - <a href="http://babelfish.altavista.digital.com/cgi-bin/translate?urltext=http%3a%2f%2fgost%2eisi%2eedu%2f%7ejohnh%2fSOFTWARE%2fAPACHE%2f&language=en">Translate</a></i><br>
<P></dl>
<font size=-1 face=arial><font size=-1>Word count: LSAM: 395; ISI: 135759; work: 23719395<BR></font>
</font><P>1 <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10">2 </a><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=20">3 </a><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=30">4 </a><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=40">5 </a><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bwork&stq=10">[Next]</a><br>
<HR><font face="arial, helvetica" size=-1>
<a href="/av/content/av_network.htm">Our Network</a> | 
<a href="/av/content/addurl.htm">Add/Remove URL</a> | 
<a href="/av/content/questions.htm">Feedback</a> | 
<a href="/av/content/help.htm">Help</a><br>

<a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | 
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/av/content/jobs.htm">Jobs</a> | 
<a href="/cgi-bin/query?pg=q">Graphics</a></font><HR>
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy Statement</a><br>

<a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1995-98. All Rights Reserved
</FORM>
</BODY></HTML>
