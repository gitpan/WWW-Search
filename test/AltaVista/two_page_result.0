<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +IB</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Ask AltaVista<sup><small>TM</small></sup> a question.</B>&nbsp;&nbsp;Or enter a few words in
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+LSAM +ISI +IB">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 <P><font size=-1>Search - <a href="/cgi-bin/query?pg=aq&what=web&text=yes">Advanced</a> - <a href="/cgi-bin/query?pg=q&what=news&text=yes">Usenet</a></font><BR></FORM>
<font size=-1><b>AltaVista found 18 Web pages for you. </font></b><P><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-1><dl><dt><b>1. </b><a href="http://www.isi.edu/ib/example.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/example.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fexample%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>2. </b><a href="http://www.isi.edu/div7/cities/ib/tuner/about.html"><b>About IB</b></a><dd>
About the Multicast Web Tuner. Proxy Benefits: Move files closer to client to improve response speed when files are requested again. Share proxies so that.<br><b>URL:</b> <font color=gray>www.isi.edu/div7/cities/ib/tuner/about.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fcities%2fib%2ftuner%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>3. </b><a href="http://www.isi.edu/~lsam/ib/rmc/investigate.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. AFDP. Dynamically selects from available transmission modes so as to maximize efficiency and throughput while..<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/investigate.html<br>
Last modified 12-Feb-98 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2finvestigate%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>4. </b><a href="http://www.isi.edu/~lsam/ib/rmc/links.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Here are a few good links for Multicast Transport Protocols....<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/links.html<br>
Last modified 12-Feb-98 - page size 1K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2flinks%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>5. </b><a href="http://www.isi.edu/ib/discussion.html"><b>IB Discussion</b></a><dd>
IB Discussion. Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web browsers, Gopher and WAIS...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/discussion.html<br>
Last modified 13-Jan-97 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2fdiscussion%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>6. </b><a href="http://www.isi.edu/div7/ib/plan.html"><b>IB Research Plan</b></a><dd>
IB Research Plan. For the experiments in this research, we will make use of traffic generators that emulate WWW browser request streams, to evaluate the...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/ib/plan.html<br>
Last modified 13-Jan-97 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2fib%2fplan%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>7. </b><a href="http://www.isi.edu/~lsam/ib/index.html"><b>About IB</b></a><dd>
Intelligent Bandwidth. About Intelligent Bandwidth. Research Projects. People. Related Work. Page maintainer: Joe Touch Last modified: Thu Jan 15 13:30:29.<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/index.html<br>
Last modified 15-Jan-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2findex%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>8. </b><a href="http://www.isi.edu/~lsam/ib/people.html"><b>IB Research Plan</b></a><dd>
IB People. Joe Touch (project leader) Amy S. Hughes. Anand Oswal. Page maintainer: Joe Touch Last modified: Thu Jan 15 13:31:54 1998 Copyright © 1996 by...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/people.html<br>
Last modified 15-Jan-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2fpeople%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>9. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/about.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Goals. To use idle bandwidth to pre-load caches with objects that might be of interest in the future. The...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/about.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>10. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/design.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism: The idea of multicasting NACks is based on the fact that multicast losses are...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/design.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2fdesign%2ehtml&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><b>Result Pages: </b><b>1</b> &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">2</a>
&nbsp; <a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=10&c9k">[<b>Next &gt;&gt;</b>]</a> <br>
</font><font size=-2>word count: LSAM: 893; ISI: 130445; IB: 169417<BR></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
