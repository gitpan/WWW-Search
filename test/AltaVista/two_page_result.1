<html><head>
<title>AltaVista: Simple Query +LSAM +ISI +IB</title>
</head><body>
<font size=+1 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Search Service</a>
</font>
<br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1><B>Ask AltaVista<sup><small>TM</small></sup> a question.</B>&nbsp;&nbsp;Or enter a few words in
<SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh>Chinese
<OPTION VALUE=cs>Czech
<OPTION VALUE=da>Danish
<OPTION VALUE=nl>Dutch
<OPTION VALUE=en>English
<OPTION VALUE=et>Estonian
<OPTION VALUE=fi>Finnish
<OPTION VALUE=fr>French
<OPTION VALUE=de>German
<OPTION VALUE=el>Greek
<OPTION VALUE=he>Hebrew
<OPTION VALUE=hu>Hungarian
<OPTION VALUE=is>Icelandic
<OPTION VALUE=it>Italian
<OPTION VALUE=ja>Japanese
<OPTION VALUE=ko>Korean
<OPTION VALUE=lv>Latvian
<OPTION VALUE=lt>Lithuanian
<OPTION VALUE=no>Norwegian
<OPTION VALUE=pl>Polish
<OPTION VALUE=pt>Portuguese
<OPTION VALUE=ro>Romanian
<OPTION VALUE=ru>Russian
<OPTION VALUE=es>Spanish
<OPTION VALUE=sv>Swedish
</SELECT></font><br><INPUT NAME=q size=50 maxlength=800 VALUE="+LSAM +ISI +IB">
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit><br>
 <P><font size=-1>Search - <a href="/cgi-bin/query?pg=aq&what=web&text=yes">Advanced</a> - <a href="/cgi-bin/query?pg=q&what=news&text=yes">Usenet</a></font><BR></FORM>
<font size=-1><b>AltaVista found 18 Web pages for you. </font></b><P><font size=-1><b>Result Pages: </b><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=0&c9k">[<b>&lt;&lt; Prev</b>]</a>  &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=0&c9k">1</a>
 <b>2</b><br>
</font><font size=-1><dl><dt><b>11. </b><a href="http://www.isi.edu/~lsam/ib/rmc/design.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism: The idea of multicasting NACks is based on the fact that multicast losses are...<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/ib/rmc/design.html<br>
Last modified 12-Feb-98 - page size 4K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fib%2frmc%2fdesign%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>12. </b><a href="http://www.isi.edu/ib/"><b>About IB</b></a><dd>
About Intelligent BW. In today's Internet, information services such as WWW, Gopher, WAIS, etc. are oriented towards a reactive model: the Internet and...<br><b>URL:</b> <font color=gray>www.isi.edu/ib/<br>
Last modified 6-Oct-97 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fib%2f&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>13. </b><a href="http://www.isi.edu/div7/lsam/ib/projects.html"><b>IB Research Plan</b></a><dd>
IB Research Projects. Lowlat. Anticipatory document caching reduces request latency. Multicast Web Tuner. Multicasting popular documents on channels...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/projects.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2fprojects%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>14. </b><a href="http://www.isi.edu/div7/lsam/ib/about.html"><b>About IB</b></a><dd>
About Intelligent BW. Motivations. Ethernet or faster bandwidth is needed to interactively view the average web page. Bandwidth requirements increase as...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/about.html<br>
Last modified 15-Jan-98 - page size 3K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2fabout%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>15. </b><a href="http://www.isi.edu/div7/lsam/ib/rmc/req.html"><b>IB - Reliable Multicast Transport protocol</b></a><dd>
IB - Reliable Multicast Transport protocol. Reliability Mechanism. Retransmission is event driven (off-line). Retransmission requests uses selactive NACKS.<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/rmc/req.html<br>
Last modified 12-Feb-98 - page size 2K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frmc%2freq%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>16. </b><a href="http://www.isi.edu/div7/lsam/ib/related-work.html"><b>LSAM Related Research</b></a><dd>
Web Latency Reduction. Features Description. Anticipation. Configuration. Cache Organization. Intermediate Proxies. User Aggregation. Connection...<br><b>URL:</b> <font color=gray>www.isi.edu/div7/lsam/ib/related-work.html<br>
Last modified 15-Jan-98 - page size 6K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2fdiv7%2flsam%2fib%2frelated%2dwork%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>17. </b><a href="http://www.darpa.mil/ito/psum1997/C370-0.html"><b>LSAM: Large-Scale Active Middleware (formerly Cities Online)</b></a><dd>
DARPA ITO Sponsored Research. 1997 Project Summary LSAM: Large-Scale Active Middleware (formerly Cities Online) USC Information Sciences Institute....<br><b>URL:</b> <font color=gray>www.darpa.mil/ito/psum1997/C370-0.html<br>
Last modified 14-Jan-99 - page size 9K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2edarpa%2emil%2fito%2fpsum1997%2fC370%2d0%2ehtml&language=en">Translate</a>&nbsp;]</dl>
<dl><dt><b>18. </b><a href="http://www.isi.edu/~lsam/about/"><b>About LSAM</b></a><dd>
About LSAM. What is LSAM. LSAM is developing middleware infrastructure to support scalable distributed information services. This is embodied in the LSAM..<br><b>URL:</b> <font color=gray>www.isi.edu/~lsam/about/<br>
Last modified 18-Feb-99 - page size 9K - in English</font> [&nbsp;<a href="http://jump.altavista.com/trans.go?urltext=http%3a%2f%2fwww%2eisi%2eedu%2f%7elsam%2fabout%2f&language=en">Translate</a>&nbsp;]</dl>
</font><font size=-1><b>Result Pages: </b><a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=0&c9k">[<b>&lt;&lt; Prev</b>]</a>  &nbsp;<a href="/cgi-bin/query?pg=q&text=yes&q=%2bLSAM+%2bISI+%2bIB&stq=0&c9k">1</a>
 <b>2</b><br>
</font><font size=-2>word count: LSAM: 893; ISI: 130445; IB: 169417<BR></font><P>
<br>
<font size=-2 face=arial, helvetica>
<a href="/cgi-bin/query?text">AltaVista Home</a> | 
<a href="/av/content/help.htm">Help</a> | 
<a href="/av/content/questions.htm">Feedback</a>
<br> 
Copyright | 
<a href="/av/content/disclaimer.htm">Disclaimer</a> | 
<a href="/av/content/privacy.htm">Privacy</a>
<br>
<a href="/av/content/about.htm">About AltaVista</a> | 
<a href="/cgi-bin/query?pg=tmpl&v=pref.html">Set your Preferences</a> | 
<a href="/av/content/addurl.htm">Add a Page</a> | 
<a href="/cgi-bin/query">Graphic Version</a>
</font>
</BODY></HTML>
