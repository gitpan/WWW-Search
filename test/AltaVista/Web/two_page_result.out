http://www.isi.edu/ib/example.html
http://www.isi.edu/div7/cities/ib/tuner/about.html
http://www.isi.edu/~lsam/ib/rmc/investigate.html
http://www.isi.edu/~lsam/ib/rmc/links.html
http://www.isi.edu/ib/discussion.html
http://www.isi.edu/div7/ib/plan.html
http://www.isi.edu/~lsam/ib/index.html
http://www.isi.edu/~lsam/ib/people.html
http://www.isi.edu/div7/lsam/ib/rmc/about.html
http://www.isi.edu/div7/lsam/ib/rmc/design.html
http://www.isi.edu/~lsam/ib/rmc/design.html
http://www.isi.edu/ib/
http://www.isi.edu/div7/lsam/ib/projects.html
http://www.isi.edu/div7/lsam/ib/about.html
http://www.isi.edu/div7/lsam/ib/rmc/req.html
http://www.isi.edu/div7/lsam/ib/related-work.html
http://www.darpa.mil/ito/psum1997/C370-0.html
http://www.isi.edu/~lsam/about/
